loopy/kernel/__init__.py

```diff
@@ -236,6 +236,10 @@ class LoopKernel(ImmutableRecordWithoutPickling):
 
         A subclass of :class:`loopy.TargetBase`.
 
+    .. attribute:: inames
+
+        An instance of :class:`frozenset` of :class:`loopy.kernel.data.Iname`.
```

Comment:
Went with a `dict` here.  (pytools doesn't support `pyrsistent` data structures in PersistentDict)

Reply:
:( OK. I'm really eager to try and rewire Loopy to use `pyrsistent` to avoid mutability accidents (and perhaps improve performance), but I guess it can wait.